.env.travis

```diff
@@ -0,0 +1,30 @@
+AWS_ACCESS_KEY_ID=
+AWS_REGION=
+AWS_SECRET_ACCESS_KEY=
+DATABASE_POOL_SIZE=15
+#POSTGRES_PASSWORD=password
+DB_ENV_POSTGRES_DATABASE=postgres
+DB_ENV_POSTGRES_PASSWORD=password
+DB_ENV_POSTGRES_USER=postgres
+DB_PORT_5432_TCP_ADDR=db
+DB_PORT_5432_TCP_PORT=5432
+DEVISE_PEPPER=
+DEVISE_SECRET_KEY=
+IA_ACCESS_KEY_ID=
+IA_SECRET_ACCESS_KEY=
+MAIL_FROM=fixer@prx.org
+MAIL_HOST=localhost
+OPEN_CALAIS_API_KEY=
+SECRET_KEY_BASE=
+SECRET_TOKEN=
+SHORYUKEN_DELAY=5
+SIDEKIQ_DATABASE_POOL_SIZE=10
+SSL_VERIFY_PEER=true
+WORKER_COUNT=10
+WORKER_LIB=shoryuken
+YAHOO_CONTENT_ANALYSIS_API_KEY=
+YAHOO_CONTENT_ANALYSIS_API_SECRET=
+# TODO LOCAL_ENV for travis?
```

Comment:
@kookster what do you think about `LOCAL_ENV` and `TRAVIS`? I disabled the `TRAVIS` and all tests pass, which makes me think we should strip all that checking code out in a subsequent PR.

Is `LOCAL_ENV` the right thing here for travis?

Reply:
`LOCAL_ENV` is the right thing for travis and developers, as it will prevent trying to load secrets from the aws env.

We used travis before we had our own cicd process, not sure we need to keep supporting it at all.
I mean, its an open source project, so yeah, I guess its nice to do so.